lm75bd/lm75bd.c

```diff
@@ -26,7 +26,17 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+  uint8_t pointerRegister = 0;
+  i2cSendTo(LM75BD_OBC_I2C_ADDR, &pointerRegister, 1);
```

Comment:
- [ ] wrap inside of the RETURN_IF_ERROR_CODE macro as we don't want to continue executing code if this fails
- [ ] use sizeof instead of harding coding a number